packages/testsuite/cypress/e2e/transaction/transactions-configuration-subsystem-transaction-configuration.cy.ts

```diff
@@ -0,0 +1,177 @@
+// navigation crumbs in the management console
+describe("TESTS: Configuration => Sybsystem => Transaction => Configuration", () => {
+  const transactions = "transactions";
+  const address = ["subsystem", transactions];
+  const configTab = "#tx-attributes-config-item";
+  const configurationForm = "tx-attributes-form";
+  const defaultTimeout = "default-timeout";
+  const enableTsmStatus = "enable-tsm-status";
+  const journalStoreEnableAIO = "journal-store-enable-async-io";
+  const jts = "jts";
+  const maximumTimeout = "maximum-timeout";
+  const nodeIdentifier = "node-identifier";
+  const statisticsEnabled = "statistics-enabled";
+  const useJurnalStore = "use-journal-store";
+
+  let managementEndpoint: string;
+
+  // Here we start our WildFly container prior to test execution
+  before(() => {
+    cy.startWildflyContainer().then((result) => {
+      managementEndpoint = result as string;
+    });
+  });
+
+  after(() => {
+    cy.task("stop:containers");
+  });
+
+  it("Edit Default Timeout", () => {
+    cy.task("execute:cli", {
+      managementApi: managementEndpoint + "/management",
+      operation: "read-attribute",
+      address: address,
+      name: defaultTimeout,
+    }).then((result) => {
+      const newValue = (result as { result: number }).result + 100;
+      cy.navigateTo(managementEndpoint, transactions);
+      cy.get(configTab).click();
+      cy.editForm(configurationForm);
+      cy.text(configurationForm, defaultTimeout, newValue);
+      cy.saveForm(configurationForm);
+      cy.verifySuccess();
+      cy.verifyAttribute(managementEndpoint, address, defaultTimeout, newValue);
+    });
+  });
+
+  it("Toggle Enable Tsm Status", () => {
+    let defaultValue = false;
+    cy.task("execute:cli", {
```

Comment:
here you can use the `cy.readAttributeAsBoolean` method and on other places where you need load boolean value also